CMakeLists.txt

```diff
@@ -40,7 +40,11 @@ file(GLOB UOSM_CORE_SOURCES
         UOSM-Core/Utils/*.cpp
 )
 
-add_executable(UOSM_Telemetry_Pico ${SOURCES} ${UOSM_CORE_SOURCES})
+add_executable(UOSM_Telemetry_Pico ${SOURCES} ${UOSM_CORE_SOURCES}
+        Core/Tasks/CurrentVoltageTask.h
+        Core/Tasks/CurrentVoltageTask.cpp
+        Core/Tasks/Accelerometer.c
```

Comment:
These .c files are done in the GLOB SOURCES at line 31
They should not be needed.

Reply:
I'll look into this. I believe CLion had automatically added these to the CMake file.